config/initializers/assets.rb

```diff
@@ -10,4 +10,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
-Rails.application.config.assets.precompile += %w[jquery.js bootstrap.js]
+Rails.application.config.assets.precompile += %w[jquery.js popper.js bootstrap.js]
```

Comment:
you said you removed jquery? or only our external dependency? does bootstrap ship its own jquery now? or did you just forget to remove it here? :)

Reply:
It was (of course) the latter :man_facepalming: 

Bootstrap really no longer needs jQuery. And neither do we.

Thanks for catching that!